CSharp/Library/Microsoft.Bot.Connector.Shared/ConnectorAPI/Models/Activity.cs

```diff
@@ -317,6 +334,19 @@ public Activity(string type = default(string), string id = default(string), Syst
         [JsonProperty(PropertyName = "replyToId")]
         public string ReplyToId { get; set; }
 
+        /// <summary>
+        /// Gets or sets descriptive label
+        /// </summary>
+        [JsonProperty(PropertyName = "label")]
+        public string Label { get; set; }
+
+        /// <summary>
+        /// Gets or sets unique string which identifies the shape of the value

```

Comment:
Same thing I think "Unique string which identifies the format of the value object"  is a better comment here